site/src/Consumer/UpdateProjectInformation.php

```diff
@@ -30,23 +33,31 @@ class UpdateProjectInformation extends AbstractProjectUpdateConsumer
      */
     private $entityManager;
 
+    /**
+     * @var NotificationCreatorService
+     */
+    private $notificationCreator;
+
     /**
      * UpdateProjectInformation constructor.
      *
-     * @param RepoCrawlerRegistry    $repoCrawlerFactory
-     * @param LoggerInterface        $logger
-     * @param ProjectRepository      $projectRepository
-     * @param EntityManagerInterface $entityManager
+     * @param RepoCrawlerRegistry        $repoCrawlerFactory
+     * @param LoggerInterface            $logger
+     * @param ProjectRepository          $projectRepository
+     * @param EntityManagerInterface     $entityManager
+     * @param NotificationCreatorService $notificationCreatorService
      */
     public function __construct(
         RepoCrawlerRegistry $repoCrawlerFactory,
         LoggerInterface $logger,
         ProjectRepository $projectRepository,
-        EntityManagerInterface $entityManager
+        EntityManagerInterface $entityManager,
+        NotificationCreatorService $notificationCreatorService
     ) {
         parent::__construct($projectRepository, $logger);
         $this->repoCrawlerRegistry = $repoCrawlerFactory;
         $this->entityManager = $entityManager;
+        $this->notificationCreator = $notificationCreatorService;
```

Comment:
Use consistent naming: call the member variable notificationCreatorService as well.